Sming/SmingCore/Platform/AccessPoint.h

```diff
@@ -70,9 +70,10 @@ class AccessPointClass : protected ISystemReadyHandler
 	bool setIP(IPAddress address);
 
 	/** @brief  Get WiFi AP MAC address
+	 *  @param optional separator between bytes (e.g. ':')
      *  @retval String WiFi AP MAC address
      */
-	String getMAC();
+	String getMAC(char sep = '\0');
```

Comment:
> as 00:11:22:aa:bb:cc, while previously they returned it as 001122aabbcc

@mikee47 Shouldn't the delimiter be an empty string `''` , instead of `'\0'` ?

Reply:
Thanks.